project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Footer headings are not visible 
<img width="1302" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/65a17883-ce79-4d59-8697-6c713cd1f19f">